### PR TITLE
Exception Student

### PR DESCRIPTION
**Pull request by katieann7**

### CODE REVIEW

File: student-information-management-application/src/test/java/com/student/information/management/app/facade/student/impl/StudentFacadeImplTest.java [hunk 1]
```diff
@@ -51,13 +51,11 @@ public void testAddStudent() {
      */
     @Test
     public void testUpdateStudent() {
-        Student existingStudent = studentFacade.getStudentById("CT21-1111");
-        existingStudent.setAddress("SanFrancisco");
-        studentFacade.updateStudent(existingStudent);
+        String studentId = "CT21-0073";
```
Comment (RoCS2024) [inline student-information-management-application/src/test/java/com/student/information/management/app/facade/student/impl/StudentFacadeImplTest.java:54]: What is the purpose of this test method?

File: student-information-management-application/src/test/java/com/student/information/management/data/student/dao/impl/StudentDaoImplTest.java [hunk 2]
```diff
@@ -84,21 +84,25 @@ public Boolean answer(InvocationOnMock invocationOnMock) throws Throwable {
...
-    private static final String TEST_STUDENT_ID = "CT21-1111";
     /**
      * This is to test Update Student.
      */
+
+
     @Test
-    void testUpdateStudent() {
+    void testUpdateStudentById() {
         Student testStudent = new Student();
```
Comment (RoCS2024) [inline student-information-management-application/src/test/java/com/student/information/management/data/student/dao/impl/StudentDaoImplTest.java:94]: why use the actual database? mock it please?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Comment by katieann7** [comment]
All done po sir.

File: student-information-management-application/src/main/java/com/student/information/management/app/model/student/Student.java [hunk 3]
```diff
@@ -21,6 +21,7 @@ public class Student {
     private String email;
     private String address;
     private String contactNumber;
+    private String name;
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/app/model/student/Student.java:24]: what is this name field for? you already have last, first and middle name already.

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Review by RoCS2024 — APPROVED** [review]

**Review by RoCS2024 — APPROVED** [review]

**Review by rosellemarithe — APPROVED** [review]
all goods, please merge it

**Pull request merged**